client/src/components/dashboard/DashboardEditor.jsx

```diff
@@ -69,6 +71,15 @@ const getFirstBlankRowGroup = (layout, height) => {
   return lastRow + 1;
 };
 
+const handleScrollToItem = (id) => {
+  scroller.scrollTo(`DashboardCanvasItem__${id}`, {
+    duration: 500,
+    smooth: true,
+    containerId: 'DashboardEditorCanvasContainer',
```

Comment:
could we extract the containerId string value to a var so it could be reused in other parts of this file?

Reply:
done